SPECS/cloud-hypervisor-cvm/cloud-hypervisor-cvm.spec

```diff
@@ -0,0 +1,229 @@
+# This spec file assumes you're building on an environment where:
```

Comment:
We should remove those comments and in generally align with cloud-hypervisor where possible